src/main/java/Duke.java

```diff
@@ -1,10 +1,423 @@
+        msg += System.lineSeparator() + "\t\tdone [task number]";
+        msg += System.lineSeparator();
+
+        // help for done command
+        msg += System.lineSeparator() + String.format("\t%s: lists the tasks currently available. " +
+                "Input must follow the format below,", CMD_LIST);
+        msg += System.lineSeparator() + "\t\tlist";
+        msg += System.lineSeparator();
+
+        // help for deadline command
+        msg += System.lineSeparator() + String.format("\t%s: adds deadline to your list of tasks. " +
+                "Input must follow the format below,", CMD_ADD_DEADLINE);
+        msg += System.lineSeparator() + "\t\tdeadline [description] /by [date/time]";
+        msg += System.lineSeparator();
+
+        // help for event command
+        msg += System.lineSeparator() + String.format("\t%s: adds event to your list of tasks. Input " +
+                "must follow the format below,", CMD_ADD_EVENT);
+        msg += System.lineSeparator() + "\t\tevent [description] /at [date/time]";
+        msg += System.lineSeparator();
+
+        // help for to do command
+        msg += System.lineSeparator() + String.format("\t%s: add todo to your list of tasks. Input" +
+                " must follow the format below,", CMD_ADD_TODO);
+        msg += System.lineSeparator() + "\t\ttodo [description]";
+        msg += System.lineSeparator();
+
+        // help for clear window
+        msg += System.lineSeparator() + String.format("\t%s: clears command window. Input" +
+                " must follow the format below,", CMD_CLEAR_WINDOW);
+        msg += System.lineSeparator() + "\t\tclear";
+        msg += System.lineSeparator();
+
+        replayBack(msg);
+    }
+
+
+    // ASCII art from: https://www.asciiart.eu/computers/bug, by Joan Start
+    private static void executeSpotBug () {
+
+        String msg = "\tNice! you found a bug :) keep it up!" + System.lineSeparator() + System.lineSeparator();
+
+        msg += "\t    .--.       .--.\n" +
+                "\t    _  `    \\     /    `  _\n" +
+                "\t     `\\.===. \\.^./ .===./`\n" +
+                "\t            \\/`\"`\\/\n" +
+                "\t         ,  | y2k |  ,\n" +
+                "\t        / `\\|;-.-'|/` \\\n" +
+                "\t       /    |::\\  |    \\\n" +
+                "\t    .-' ,-'`|:::; |`'-, '-.\n" +
+                "\t        |   |::::\\|   | \n" +
+                "\t        |   |::::;|   |\n" +
+                "\t        |   \\:::://   |\n" +
+                "\t        |    `.://'   |\n" +
+                "\tjgs    .'             `.\n" +
+                "\t    _,'                 `,_";
+
+        replayBack(msg);
+    }
+
+
+    /**
+     * Checks the type of command received, if any. Redirects
+     * path to specific execute function to process the user's input
+     *
+     * @param userResponse Raw input as entered by the user
+     */
+    private static void processUserInput(String userResponse) {
```

Comment:
Clever function to be able to differentiate between different commands

Reply:
Thank you for the feedback!